Makefile

```diff
@@ -1,10 +1,10 @@
 .PHONY: init ci analyze build rebuild migrate lang-make lang-compile
 
 init:
-	pip install pipenv --upgrade
-	pipenv install --dev --skip-lock
+	curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
```

Comment:
Just FYI, we can lock the version for get-poetry.py to its March 28 commit. 
`https://raw.githubusercontent.com/python-poetry/poetry/2f8b67e7a8cc778820554c550e8ff707f42077dd/get-poetry.py`

However the script itself will "Downloads the latest stable (or pre-release) version of poetry." so it might be a wasted effort to version-lock this line.

Reply:
Why would we want to pin the version of poetry?

Reply:
Pinning related libs/tools allows us to freeze release that covers every aspect of this project allows us to have releases to behave as expected. Our usage of poetry here now might change if they dynamically update the install script in the future. Small matter honestly, just wanted to share my research as FYI.

Reply:
I think you have a point. I remember `pipenv` breaking in the past inbetween updates and I would not want this to happen with `poetry` as well.

Reply:
That's right I remeber those times but as @am1ru1 said 

> However the script itself will "Downloads the latest stable (or pre-release) version of poetry." so it might be a wasted effort to version-lock this line.

There currently is no way to actually lock the poetry version with the recommended installation process. At least as far as I can see.